src/common/UserAvatarWithPresence.js

```diff
@@ -12,8 +12,8 @@ import PresenceStatusIndicator from './PresenceStatusIndicator';
 
 const styles = createStyleSheet({
   status: {
-    bottom: 0,
-    right: 0,
+    bottom: 4,
+    right: 4,
```

Comment:
```diff
-    bottom: 0,
-    right: 0,
+    bottom: 4,
+    right: 4,
```

Hmm, what's going on here? I haven't yet run the app with this revision, but it's not totally clear how this small code change contributes to the behavior change described in the commit message.

Reply:
this changes helps us to centralize the status indicator on the white background